NitroxClient/Communication/MultiplayerSession/MultiplayerSessionManager.cs

```diff
@@ -116,24 +118,48 @@ public void Disconnect()
             }
         }
 
-        public bool Send(Packet packet)
+        public void Send<T>(T packet) where T : Packet
+        {
+            Client.Send(packet);
+        }
+        
+        public bool SendIfGameCode<T>(T packet) where T : Packet
```

Comment:
Why does it need to be generic ?

Reply:
It doesn't need to be. But it does allow you to specify the type while sending the packet like this:
```cs
packetSender.Send<ServerStopped>(new());
```